ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/AccountTrieNodeDataRequest.java

```diff
@@ -43,7 +43,10 @@ protected void doPersist(final Updater updater) {
 
   @Override
   public Optional<Bytes> getExistingData(final WorldStateStorage worldStateStorage) {
-    return worldStateStorage.getAccountTrieNodeData(getLocation().orElse(Bytes.EMPTY), getHash());
+    return worldStateStorage
```

Comment:
if I understand well, we can store empty accounts, and if yes why?

Reply:
Bytes.EMPTY is for the location of the node in the trie. And this method return empty we there is nothing in the database. so we cannot store empty account. It's only empty when we cannot find a node in the database